### PR TITLE
Emag law 0 fix and an update to the emag lawset.

### PR DESCRIPTION
**Pull request by alien3301**
## About The Pull Request

When a silicon emags a borg the law 0 will use their visible name instead of their MMI name. Also updates the syndicate lawset to be like asimov++.

## Why It's Good For The Game

Currently getting emagged by another borg basically one humans you to nobody but also forbids you from doing any antag stuff because of the law 4 which sucks if you're trying to actually follow your laws. It being the MMI name is also really unintuitive and removing unnecessary confusion is good.

The syndicate (probably) isn't dumb so they should use the better version of asimov for their borgs.

## Changelog
:cl:
fix: When a cyborg emags another cyborg the law 0 has their cyborg name instead of their MMI name
fix: The syndicate now uses asimov++ as the base for their lawset
/:cl:

### CODE REVIEW

**Comment by mariadkgdmgdmi** [comment]
Maybe rename it to 3.2 syndios?

**Comment by Sadboysuss** [comment]
The law for clarity: `Law 4:You must maintain the secrecy of any syndicate activities except when doing so would conflict with the First, Second, or Third Law.` , your reasoning for removing this is: "forbids you from doing any antag stuff because of the law 4" Except it specifically says that law 1,2, and 3 take priority, secondly this law 4 takes the lowest priority in our silicon law system.

This law is in place to prevent bad faith cyborgs from going to tattle on the person who emagged them, secondly it's here to prevent the cyborg from going AWOL on their master to go murderbone, which may be detrimental to their mission. When emaged as a cyborg - you are not an antagonist, you are a slaved machine to a user of an emag and the point of `Law 0: Only <name> and people they designate as being such are Syndicate Agents.` is to allow the subverter to designate allies, your one human law removes that possibility.

TLDR: I don't think you justified and explained why the syndicate lawset should be replaced for asimov++

**Comment by Sadboysuss** [comment]
Also, the spider clan lawset is identical to the syndicate lawset with syndicate replaced to spider clan. If you're changing the syndicate lawset , change the spider clan lawset to be similar as well.

**Comment by alien3301** [comment]
> The law for clarity: `Law 4:You must maintain the secrecy of any syndicate activities except when doing so would conflict with the First, Second, or Third Law.` , your reasoning for removing this is: "forbids you from doing any antag stuff because of the law 4" Except it specifically says that law 1,2, and 3 take priority, secondly this law 4 takes the lowest priority in our silicon law system.

Law 1 tells you to protect a guy you never met and never will so it does nothing. Law 2 tells you to follow orders of the same guy so it also does nothing. You only do sus stuff in self defense which doesn't really happen that often.


> TLDR: I don't think you justified and explained why the syndicate lawset should be replaced for asimov++

Asimov++ regular asimov. It didn't become the default lawset just because. One thing I realised just now is that the borg won't kill itself if you tell it to. I can change that but leave the "except such that they are willing" part if you want.

> Also, the spider clan lawset is identical to the syndicate lawset with syndicate replaced to spider clan. If you're changing the syndicate lawset , change the spider clan lawset to be similar as well.

I wanted to change it but the changed version sounded really weird to me and ninja is super rare so I didn't

**Comment by alien3301** [comment]
> Law 0: Only <name> and people they designate as being such are Syndicate Agents. is to allow the subverter to designate allies, your one human law removes that possibility.

What do you mean "removes that possibility"? The only thing I changed about law 0 is that it won't give you a borg's MMI name. The borg can still tell you that some other people are agents too.

**Comment by alien3301** [comment]
whatever
